schemas/configuration/designImport.json

```diff
@@ -1,79 +1,130 @@
+  "type": "object",
   "properties": {
     "name": {
       "type": "string",
-      "title": "Give your design file a name",
+      "title": "Design file name",
       "default": "Untitled Design",
-      "x-rjsf-grid-area" : "12"
+      "x-rjsf-grid-area": "12",
+      "description": "Provide a name for your design file. This name will help you identify the file more easily. You can also change the name of your design after importing it."
     },
     "designType": {
-      "title": "Select the Design Type",
+      "title": "Select the design type",
       "enum": [
         "Helm Chart",
         "Kubernetes Manifest",
         "Docker Compose",
-        "Design"
+        "Meshery Design"
       ],
-      "default": "Design",
-      "x-rjsf-grid-area" : "12"
-    },
-    "uploadType": {
-      "title": "Select the Upload Method",
-      "enum": [
-        "File Upload",
-        "URL Upload"
-      ],
-      "default": "File Upload",
-      "x-rjsf-grid-area" : "12"
+      "default": "Meshery Design",
+      "x-rjsf-grid-area": "12",
+      "description": "Select the type of design you are uploading. The 'Design Type' determines the format, structure, and content of the file you are uploading. Choose the appropriate design type that matches the nature of your file. Checkout https://docs.meshery.io/guides/meshery-design to learn more about designs"
```

Comment:
Will the link use our standard styling?

Reply:
Yes it is @theBeginner86 

![Screenshot 2023-11-09 at 12 20 27 AM](https://github.com/meshery/meshkit/assets/71271069/142a09ae-f7ce-42ae-9891-58244e87f55c)